regex-assembly/include/unix-shell-upto3.ra

```diff
@@ -1,12 +1,18 @@
 ##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regex_assembly/.
 
-##! This list was generated from the original lists with the following command:
-##! awk '/^([a-z0-9_-]+)/ { x = $0; gsub(/[@~]/, "", x); if (length(x) <= 3) print x }' 93210{0,5}.data
+##! To regenerate this list from unix-shell.data, run the following:
+##! sed -i '' -e '14,$d' regex-assembly/include/unix-shell-upto3.ra && \
```

Comment:
Fixed.

Reply:
I think we should not be working line numbers.

If we add a comment line above the sed, completely unrelated to sed, then the sed command breaks since the line numbers are out of sync.

How about comments containing a marker?

Reply:
I already fixed all of those with dynamic determination of where the body starts.